crypto/evp/e_chacha20_poly1305.c

```diff
@@ -14,9 +14,9 @@
 
 # include <openssl/evp.h>
 # include <openssl/objects.h>
-# include "evp_locl.h"
 # include "internal/evp_int.h"
 # include "internal/chacha.h"
+# include "internal/poly1305.h"
```

Comment:
Why is this needed?

Reply:
There is a reference to the Poly block size.

Reply:
Yes?  Thing is, there is already an inclusion of `internal/poly1305.h`.  See line 147.

Reply:
Sorry I must be going crazy... removed.

Reply:
You're human, like the rest of us ;-)

Reply:
Haha, as human as I can be 😉